java/test/jmri/jmrit/display/layoutEditor/LayoutSlipTest.java

```diff
@@ -18,424 +17,455 @@
  */
 public class LayoutSlipTest {
 
-    private LayoutEditor layoutEditor = new LayoutEditor();
-    private LayoutSlip lts = new LayoutSlip("single", new Point2D.Double(50.0, 100.0), +45.0, layoutEditor, LayoutTurnout.SINGLE_SLIP);
-    private LayoutSlip ltd = new LayoutSlip("double", new Point2D.Double(100.0, 50.0), -45.0, layoutEditor, LayoutTurnout.DOUBLE_SLIP);
+    private LayoutEditor layoutEditor = null;
+    private LayoutSlip lts = null;
+    private LayoutSlip ltd = null;
 
     @Test
     public void testNew() {
-        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
```

Comment:
Leave the Assume statements in. They cause the test to be listed as "skipped" in headless mode, as opposed to be listed as "passed" in headless mode.